resources-service/src/import/importRoutesV0.ts

```diff
@@ -18,19 +18,32 @@ import { IRouter, Router } from 'express';
 import { ImportRequestBody } from '@tech-matters/types';
 import importService, { isValidationFailure } from './importService';
 import { AccountSID } from '@tech-matters/twilio-worker-auth';
+import createError from 'http-errors';
 
 const importRoutes = () => {
   const router: IRouter = Router();
 
-  const { upsertResources } = importService();
+  const { upsertResources, readImportProgress } = importService();
 
   router.post('/import', async ({ body, accountSid }, res) => {
     const { importedResources, batch }: ImportRequestBody = body;
     const result = await upsertResources(accountSid as AccountSID, importedResources, batch);
     if (isValidationFailure(result)) {
       res.status(400).json(result);
+    } else {
+      res.json(result);
+    }
+  });
+
+  router.get('/import/progress', async ({ accountSid }, res) => {
+    const progress = await readImportProgress(accountSid as AccountSID);
+    if (progress) {
+      res.json(progress);
+    } else {
+      throw createError(404, {
+        message: "No import progress found, it's possible that no import has been started yet.",
+      });
```

Comment:
How we'll start the import if the first time this will error out? I thought this endpoint was designed for the "producer" of resources.

Reply:
404s would be expected by the lambda that calls this endpoint. If a 404 is received, assume an import has never been run & call the KHP API accordingly